tests/LanguageServerCompletion/Unit/Handler/CompletionHandlerTest.php

```diff
@@ -75,8 +77,8 @@ public function testHandleSuggestions()
         );
         $this->assertInstanceOf(CompletionList::class, $response->result);
         $this->assertEquals([
-            new CompletionItem('hello'),
-            new CompletionItem('goodbye'),
+            new CompletionItem('hello', null, null, null, null, null, 'hello', null, null, null, null, 1),
+            new CompletionItem('goodbye', null, null, null, null, null, 'goodbye', null, null, null, null, 1),
```

Comment:
Tempted to suggest using something like:

```
Invoke::new(CompletionItem::class, [ 'label' => 'goodbye', 'textEdit' => 'goodbye'])
```
https://packagist.org/packages/dantleech/invoke
(actually already used by the language server)

Reply:
Ugly but don't mind for test purposes :)